lib/cdo/pardot.rb

```diff
@@ -143,8 +143,8 @@ def self.sync_new_contacts_with_pardot
       # Temporarily limit the accounts synced into Pardot to just those in
       # Teacher role, which is about 25% of total contacts. We will need to
       # purchase additional Pardot capacity to be able import all contacts.
-      where_clause: "pardot_sync_at is NULL AND pardot_id IS NULL AND "\
-                    "Roles like '%Teacher%'",
+      where_clause: "pardot_sync_at IS NULL AND pardot_id IS NULL AND "\
+                    "opted_out IS NULL",
```

Comment:
Please update (or remove?) the comment that still refers to the 25% teacher limit